docs/releases/4.1.txt

```diff
@@ -267,6 +267,19 @@ Forms
   as the template implements ``<fieldset>`` and ``<legend>`` to group related
   inputs and is easier for screen reader users to navigate.
 
+  The div-based rendererring will become the default rendering style from Django
+  5.0.
+
+* In order to smooth adopting the new ``div.html`` form template, the new
+  ``django.forms.renderers.DivRenderer`` can be set as the value for
```

Comment:
Maybe I misunderstood the intent here. When working through this I came up with a Django and a Jinja2 version. However, the docs here refer to a generic one. 🤔

Reply:
No, that's just missing.